src/core/ext/filters/client_channel/lb_policy/ring_hash/ring_hash.cc

```diff
@@ -360,12 +360,12 @@ RingHash::PickResult RingHash::Picker::Pick(PickArgs args) {
   // Ported from https://github.com/RJ/ketama/blob/master/libketama/ketama.c
   // (ketama_get_server) NOTE: The algorithm depends on using signed integers
   // for lowp, highp, and first_index. Do not change them!
-  size_t lowp = 0;
-  size_t highp = ring.size();
-  size_t first_index = 0;
+  int64_t lowp = 0;
+  int64_t highp = ring.size();
```

Comment:
you might need a static_cast<int64_t> here to avoid warnings

Reply:
That wasn't there prior to #29872, so I suspect it's not needed.